LayoutTests/fast/dynamic/anchor-lock.html

```diff
@@ -19,7 +19,10 @@ <h1>Anchor Lock</h1>
 
 if (window.testRunner) {
     testRunner.waitUntilDone();
-    window.addEventListener('scroll', () => {
+    Promise.all([
+        new Promise((resolve) => addEventListener('scroll', resolve)),
+        new Promise((resolve) => addEventListener('load', resolve)),
```

Comment:
Should we use `{ once: true }` as 3rd addEventListener option?

Reply:
I suppose it doesn't hurt, but invoking promise callbacks multiple times will result in no-ops (after the first call), so the fix should be fine as it currently is.

Reply:
I'd land this PR as for now since this is a speculative fix to begin with.